Makefile

```diff
@@ -357,8 +356,8 @@ stop:
 $(GOPATH)/bin/goreman:
 	go get github.com/mattn/goreman
 
-.PHONY: start-aux
-start-aux: $(GOPATH)/bin/goreman
+.PHONY: start
+start: status stop install controller cli executor-image $(GOPATH)/bin/goreman
```

Comment:
because `start-aux` now blocks, we do not run `wait` or `env`

Reply:
Great! This is better imo